package.json

```diff
@@ -8,8 +8,8 @@
     "build": "tsc && npm run package ",
     "format": "prettier --write **/*.ts",
     "format-check": "prettier --check **/*.ts",
-    "lint": "eslint src/**/*.ts",
-    "package": "ncc build --source-map --license license.txt",
+    "lint": "eslint src/**/*.ts --fix",
+    "package": "npx ncc build ./src/index.ts",
```

Comment:
??

Reply:
removed npx, was following the docs too closely